kobo/client/commands/cmd_create_worker.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+
+
+import sys
+from six.moves.xmlrpc_client import Fault
+
+from kobo.client import ClientCommand
+
+
+class Create_Worker(ClientCommand):
+    """create a worker"""
+    enabled = True
+    admin = True
+
+
+    def options(self):
+        self.parser.usage = "%%prog %s [--create] worker_name [worker_name]" % self.normalized_name
+
+        self.parser.add_option(
+            "--create",
+            default=False,
+            action="store_true",
+            help="create worker"
+        )
+
+
+    def run(self, *args, **kwargs):
+        create = kwargs.pop("create", False)
+
+        if len(args) == 0:
```

Comment:
This is more Pythonic:
```suggestion
        if not args:
```